types/panolens/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "dependencies": {
+        "three": "*"
```

Comment:
```suggestion
        "three": "^0.*"
```
https://github.com/pchen66/panolens.js/blob/master/package.json#L128
otherwise it would brake with v1 automatically, wouldn't it?